roles/kubernetes-apps/ansible/templates/coredns-autoscaler.yml.j2

```diff
@@ -0,0 +1,72 @@
+---
+# Copyright 2016 The Kubernetes Authors.
```

Comment:
Thanks for doing this work. 

Could this file be renamed to "dns-autoscaler.yml.j2", and be used for both kubedns and coredns?